code/modules/projectiles/guns/projectile/improvised.dm

```diff
@@ -17,6 +17,15 @@
 	origin_tech = "combat=2;materials=2"
 	handle_casings = CYCLE_CASINGS
 	load_method = SINGLE_CASING
+	
+	special_check(var/mob/living/carbon/human/M)

```

Comment:
Bad method naming convention.
`/obj/item/weapon/gun/projectile/shotgun/improvised/special_check(var/mob/living/carbon/human/M)`

^That needs to be used instead.